ep/build-velox/src/get_velox.sh

```diff
@@ -178,6 +179,8 @@ function apply_compilation_fixes {
   cd ${velox_home}
   echo "Applying patch to Velox source code..."
   git apply modify_velox.patch
+  git apply gluten_patch/*patch
+  git add .
```

Comment:
Can we consider using [`git am`](https://git-scm.com/docs/git-am) rather than `git apply` here to help a developer regenerate the patches after a manual rebase?

Reply:
Will not be feeling strongly between the two ways except we can have the patch list synced to a forked Velox repo daily to create a view of repo for the change code. That way we can retain all the required changed into one repo but does add complexity.

Reply:
Agree, unless we found a way to auto sync and generate the patch files, otherwise I can image once this PR merged, developers would suffer painful rebase / patch generation work.